src/open_inwoner/scss/components/Header/PrimaryNavigation.scss

```diff
@@ -263,12 +269,13 @@
     &:focus,
     &:focus-visible,
     &:focus-within {
+      border: 2px solid orange;
```

Comment:
@jiromaykin misschien moeten we een 'orange' pre-commit githook voor je optuigen :P gefixed in develop aangezien ik er vanuit ga dat dit niet de bedoeling was

Reply:
@alextreme dit element heeft een focus-kleur nodig voor toegankelijkheid, maar we hebben daar geen style voor; dus nieuw issue: standaard browser-focuskleur hier in terug brengen of een kleur zetten voor het hele project.

Reply:
@jiromaykin Volgens mij hebben we al een focus-kleur, toch? Ik zie 'm in ieder geval wel als ik door de site heen tab. Prima als hiervoor ook op dit element iets wordt toegevoegd, maar dan graag net zoals bij de andere elementen.

![image](https://github.com/maykinmedia/open-inwoner/assets/226839/512696d6-b7b0-41e5-9240-8120a4119306)